src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -62,10 +63,10 @@ public ErrorResponse handleResourceExists(ResourceExistsException e) {
         return produceErrorResponse(e.getMessage(), HttpStatus.CONFLICT);
     }
 
-    @ExceptionHandler(OsiamBackendFailureException.class)
+    @ExceptionHandler({OsiamBackendFailureException.class, ConnectionInitializationException.class})
```

Comment:
Why do you handle `ConnectionInitializationException` if you already catch this exception already [here](https://github.com/osiam/resource-server/pull/80/files#diff-d6745aa5dd2c5d3da174159c4d45c65dR126)?